Content.Shared/GameObjects/EntitySystems/ActionBlocker/ActionBlockerSystem.cs

```diff
@@ -24,8 +26,13 @@ public static bool CanMove(IEntity entity)
             return canMove;
         }
 
-        public static bool CanInteract(IEntity entity)
+        public static bool CanInteract([NotNullWhen(true)] IEntity? entity)
```

Comment:
Also dis.

Reply:
I think this is okay, marking as resolved for now.